tests/testsuite/cargo_config.rs

```diff
@@ -473,22 +473,20 @@ fn includes() {
         .with_stderr("")
         .run();
 
-    cargo_process(
-        "config get build.rustflags --show-origin=yes -Zunstable-options -Zconfig-include",
-    )
-    .cwd(&sub_folder.parent().unwrap())
-    .masquerade_as_nightly_cargo()
-    .with_stdout(
-        "\
+    cargo_process("config get build.rustflags --show-origin -Zunstable-options -Zconfig-include")
+        .cwd(&sub_folder.parent().unwrap())
+        .masquerade_as_nightly_cargo()
```

Comment:
>  One parser change found by `cargo_config::includes` is that clap 2
  would ignore any values after a `=` for flags.
  `cargo config --show-origin` is a flag but the test passed `--show-origin=yes` which
  happens to give the desired result for that test but is the same as
  `--show-origin=no` or `--show-origin=alien-invasion`.